### PR TITLE
PIM-5645: Removing occurences of Akeneo Native Connector

### PR DESCRIPTION
**Pull request by SamirBoulil**

### CODE REVIEW

**Comment by juliensnz** [comment]
GTM

File: reference/import_export/connectors.rst [hunk 1]
```diff
@@ -1,15 +1,15 @@
-Akeneo Native Connector
+Akeneo Connectors
 =======================
 
-CSV Format
+CSV Connector
 ----------
```
Comment (phaseinducer) [inline reference/import_export/connectors.rst]: Please correct the length

File: reference/import_export/connectors.rst [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-Akeneo Native Connector supports reading and writing CSV files.
+Akeneo CSV Connector supports reading and writing CSV files.
 
-XLSX Format
+XLSX Connector
 -----------
```
Comment (phaseinducer) [inline reference/import_export/connectors.rst]: idem

File: reference/import_export/connectors.rst [hunk 3]
```diff
@@ -23,7 +23,7 @@ The connector supports the following Excel versions:
 Customizing connectors
 ----------------------
 
-It is possible to customize this connector to import and export numerous PIM related objects:
+It is possible to customize these connectors to import and export numerous PIM related objects:
 
-- Check how to import PIM objects using this connector (cf :doc:`reference/import_export/simple-import.rst`)
-- In order to understand how the connector is working for products, please refer to chapters about product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`)
+- Check how to import PIM objects using the Akeneo Connectors (cf :doc:`/reference/import_export/simple-import`)
+- In order to understand how the Akeneo CSV Connector is working for products, please refer to chapters about product import (cf :doc:`/reference/import_export/product-export`) and product export (cf :doc:`/reference/import_export/product-export`)
```
Comment (phaseinducer) [inline reference/import_export/connectors.rst]: Why just CSV?

Comment (SamirBoulil) [inline reference/import_export/connectors.rst]: Because the documentation references explain how the CSV connector is working.

**Comment by pierallard** [comment]
GTM

**Comment by rybus** [comment]
GTM

**Pull request merged**